### PR TITLE
fix(navigation): move Page html demos to Navigation component

### LINKED ISSUE #5106: Move "Page" html demos into "Navigation" component
Change `id: Page` to `id: Navigation`.

Related https://github.com/patternfly/patternfly/issues/3447

### PR DESCRIPTION
**Pull request by boaz0**
**What**: Closes #5106 

// cc @redallen

### CODE REVIEW

**Review by redallen — APPROVED** [review]
Thanks @boaz0 !

**Review by kmcfaul — APPROVED** [review]

**Review by tlabaj — CHANGES_REQUESTED** [review]
I am a little confused with this PR.  where are the Nav props documented?

File: packages/react-core/src/components/Page/examples/Page.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-id: Page
+id: Navigation
```
Comment (redallen) [inline packages/react-core/src/components/Page/examples/Page.md:2]: Revert this.

File: packages/react-core/src/demos/Page/Page.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-id: Page
+id: Navigation
 section: components
 ---
```
Comment (redallen) [inline packages/react-core/src/demos/Page/Page.md:4]: Move demos that include navigation into a new `demos/Nav/Nav.md` file with:
```
---
id: Navigation
section: components
---
```

**Review by redallen — CHANGES_REQUESTED** [review]
Sorry I didn't take a closer look first pass-through. I think this is what design is asking for.

File: packages/react-core/src/demos/Page/Page.md (hunk 2 above)
Comment (redallen) [reply packages/react-core/src/demos/Page/Page.md:4]: These are the demos that should be moved:
Default nav
Expandable nav
Horizontal nav
Tertiary nav
Grouped nav

File: packages/react-core/src/components/Page/examples/Page.md (hunk 1 above)
Comment (boaz0) [reply packages/react-core/src/components/Page/examples/Page.md:2]: Oh, my mistake.

File: packages/react-core/src/demos/Page/Page.md (hunk 2 above)
Comment (boaz0) [reply packages/react-core/src/demos/Page/Page.md:4]: Thanks.

File: packages/react-core/src/components/Nav/examples/Nav.md [hunk 3]
```diff
@@ -457,3 +457,1155 @@ class NavTertiaryList extends React.Component {
   }
 }
 ```
+
+### Default nav
```
Comment (redallen) [inline packages/react-core/src/components/Nav/examples/Nav.md:461]: Super close! Just move all these demos to a new `react-core/src/demos/Nav.md` file.

**Review by redallen — CHANGES_REQUESTED** [review]

File: packages/react-core/src/demos/Page/Page.md [hunk 4]
```diff
@@ -23,1157 +23,6 @@ import imgAvatar from '@patternfly/react-core/src/components/Avatar/examples/ava
...
-        </Page>
-      </React.Fragment>
-    );
-  }
-}
-```
-
 ### Sticky section group
 
 ```js isFullscreen
```
Comment (redallen) [inline packages/react-core/src/demos/Page/Page.md:28]: Can you move the last 2 examples too?
- Grouped nav
- Light themed sidebar and nav

**Review by redallen — CHANGES_REQUESTED** [review]

File: packages/react-core/src/demos/Page/Page.md (hunk 4 above)
Comment (boaz0) [reply packages/react-core/src/demos/Page/Page.md:28]: Grouped nav is already in demos/Nav.md, isn't it?

Comment (redallen) [reply packages/react-core/src/demos/Page/Page.md:28]: Ah sorry, I meant "Manual nav". That's the last one to move to the page demos and then this looks good!

Comment (boaz0) [reply packages/react-core/src/demos/Page/Page.md:28]: done :+1:

**Review by redallen — APPROVED** [review]
Thanks again @boaz0 !

**Comment by boaz0** [comment]
no problem. I am sorry it took me some time to understand what needs to be done though :smiley:  :octopus:

**Review by tlabaj — APPROVED** [review]

**Pull request merged**